WooCommerce/Classes/ViewRelated/Orders/Order Creation/FlowCoordinator/NewSimplePaymentsLocationNoticeViewModel.swift

```diff
@@ -0,0 +1,40 @@
+import SwiftUI
+
+final class NewSimplePaymentsLocationNoticeViewModel {
+    /// Redirects to `HubMenu`tabBar
+    ///
+    var navigateToMenuButtonWasTapped: () -> Void = {
+        guard let mainTabBarController = AppDelegate.shared.tabBarController else {
+            return
+        }
+        mainTabBarController.navigateTo(.hubMenu)
+    }
+    /// Uses the "woo-payments-wallet" UIImage+Woo as icon
+    ///
+    var icon: UIImage {
+        guard let image = UIImage(named: "woo-payments-wallet") else { return UIImage() }
```

Comment:
You can use here the `static var walletImage: UIImage ` in UIImage+Woo, and avoid the guard

Reply:
That makes sense, changed at 40d089712c45a0bfb74ad2298cddc5f5403212c0